src/config/path-util.ts

```diff
@@ -35,4 +35,4 @@ class PathStore {
   }
 }
 
-export default new PathStore()
+export const pathUtil = new PathUtil()
```

Comment:
A couple of default exports were changed to named exports so they can be exported from the module in a consistent way.